GCC 10.0.1 compile error

<!-- Please help us help you: We normally need the information below in order to answer questions or understand bug reports. If you do not provide the information requested we may not be able to help you and will probably close the issue. --> 

**Configuration:**

<!-- Information about your setup. -->
<!-- To get the RDKit version use the following commands:
import rdkit
rdkit.__version__
-->
- RDKit Version: 2020.03.1, and also the current tip of the master branch
- Operating system: linux (Fedora 32)
- Python version (if relevant): not relevant
- Are you using conda?
- If you are using conda, which channel did you install the rdkit from?
- If you are not using conda: how did you install the RDKit?

**Description:**

<!-- Please describe your problem/question in as much detail as possible -->
The new Fedora 32 release upgrades the default gcc compilers to version 10.0.1. A build of the RDKit C++ libraries fails with the following error:
```
/Code/GraphMol/catch_sgroups.cpp:210:6: error: could not convert ‘{{{{{{1.0e+0, 3.0e+0, 0.0}}, {{5.0e+0, 7.0e+0, 0.0}}, {{0.0, 0.0, 0.0}}}}, {{{{2.0e+0, 4.0e+0, 0.0}}, {{6.0e+0, 8.0e+0, 0.0}}, {{0.0, 0.0, 0.0}}}}}}’ from ‘<brace-enclosed initializer list>’ to ‘std::vector<std::array<std::array<double, 3>, 3> >’
  210 |     }};
      |      ^
      |      |
      |      <brace-enclosed initializer list>
```
I'm not sure if it's the proper fix, but, replacing the outermost double braces with single ones, apparently fixes the compile-time error. A similar fix also applies in two locations inside `/Code/GraphMol/testSGroup.cpp`.

<!-- If you have a support question, please use the rdkit-discuss mailing list: 
https://sourceforge.net/p/rdkit/mailman/ -->

--- a/Code/GraphMol/catch_sgroups.cpp
+++ b/Code/GraphMol/catch_sgroups.cpp
@@ -204,10 +204,10 @@ TEST_CASE("Build and test sample molecule", "[Sgroups]") {
     CHECK(sg.getProp<unsigned int>("COMPNO") == 7u);
     CHECK(sg.getProp<std::string>("ESTATE") == "E");
 
-    std::vector<std::array<std::array<double, 3>, 3>> brackets_reference = {{
+    std::vector<std::array<std::array<double, 3>, 3>> brackets_reference = {
         {{{{1., 3., 0.}}, {{5., 7., 0.}}, {{0., 0., 0.}}}},
         {{{{2., 4., 0.}}, {{6., 8., 0.}}, {{0., 0., 0.}}}},
-    }};
+    };
     testBrackets(sg.getBrackets(), brackets_reference);
 
     auto cstates = sg.getCStates();

--- a/Code/GraphMol/testSGroup.cpp
+++ b/Code/GraphMol/testSGroup.cpp
@@ -192,10 +192,10 @@ void checkSampleMolecule(const RWMol &mol) {
     TEST_ASSERT(sg.getProp<unsigned int>("COMPNO") == 7);
     TEST_ASSERT(sg.getProp<std::string>("ESTATE") == "E");
 
-    std::vector<std::array<std::array<double, 3>, 3>> brackets_reference = {{
+    std::vector<std::array<std::array<double, 3>, 3>> brackets_reference = {
         {{{{1., 3., 0.}}, {{5., 7., 0.}}, {{0., 0., 0.}}}},
         {{{{2., 4., 0.}}, {{6., 8., 0.}}, {{0., 0., 0.}}}},
-    }};
+    };
     testBrackets(sg.getBrackets(), brackets_reference);
 
     auto cstates = sg.getCStates();
@@ -324,10 +324,10 @@ void testParseSubstanceGroups(const std::string &rdbase) {
         {};  // No bonds defined in this mol
     testIdxVector(sgroup.getBonds(), bonds_reference);
 
-    std::vector<std::array<std::array<double, 3>, 3>> brackets_reference = {{
+    std::vector<std::array<std::array<double, 3>, 3>> brackets_reference = {
         {{{{-3.9679, -0.1670, 0.}}, {{-3.9679, 2.1705, 0.}}, {{0., 0., 0.}}}},
         {{{{-0.7244, 2.1705, 0.}}, {{-0.7244, -0.1670, 0.}}, {{0., 0., 0.}}}},
-    }};
+    };
     testBrackets(sgroup.getBrackets(), brackets_reference);
   }
 

